libs/cli/src/main/java/org/elasticsearch/cli/ExitCodes.java

```diff
@@ -13,6 +13,7 @@
  */
 public class ExitCodes {
     public static final int OK = 0;
+    public static final int NOOP = 63;           /* nothing to do */
```

Comment:
Am I missing something? This doesn't seem to be used anywhere.

Reply:
This is a draft PR and the change above was from  #74868. I appreciate the early feedback but I think it's better use of your folks time if we review this PR when it's ready

Reply:
> I appreciate the early feedback but I think it's better use of your folks time if we review this PR when it's ready

No worries. I have no problems giving draft feedback, and if it's something that you all intend to already address that's all good. I'll defer any further review until you give the go ahead.